src/ADNLPProblems/trig.jl

```diff
@@ -0,0 +1,20 @@
+export trig
+
+function trig(; n::Int = default_nvar, type::Type{T} = Float64, kwargs...) where {T}
+  function f(x; n = length(x))
+    s = zero(T)
+    for i = 1:n
+      s += i * (1 - cos(x[i]))
+
+      for j = max(1, i - 2):min(n, i + 2)
+        aij = 5 * (1 + mod(i, 5) + mod(j, 5))
+        bij = (i + j) / 10
```

Comment:
@arnavk23 My guess is that this is your problem `(i + j) / 10` is a Float64, you should have `(i + j) // 10`